src/components/settings/AddonsTab.vue

```diff
@@ -96,11 +99,14 @@ export default {
     credentialItems: function (newValue) {
       this.credentials = newValue;
     },
+    configItem: function (newValue) {
+      this.config = newValue;
+    },
   },
   computed: {
-    ...mapGetters({
-      config: "config/fullConfig",
-    }),
+    // ...mapGetters({
+    //   config: "config/fullConfig",
+    // }),
```

Comment:
@dockabernathy01 you reverted this to the "old style" of passing configs.  Please see how all of the other `settings` tabs handle this and store the data in state rather than via props.  These props like `configItem` are exactly the sort of thing that should be getting removed in this issue: https://github.com/yatt-ai/yattie/issues/147

So please revert these changes around props and getters.

Reply:
Ok, got it.